python/ray/train/batch_predictor.py

```diff
@@ -150,6 +125,40 @@ def predict(
         Returns:
             Dataset containing scoring results.
 
+        Examples:
+
+            .. testcode::
+
+                import pandas as pd
+                import ray
+                from ray.air import Checkpoint
+                from ray.train.predictor import Predictor
+                from ray.train.batch_predictor import BatchPredictor
+
+                def calculate_accuracy(df):
+                    return pd.DataFrame({"correct": df["preds"] == df["label"]})
```

Comment:
I believe this needs to be `predictions` instead of `preds` :)

Reply:
Yes, I changed it to fit the lint limit of 88 characters. So we have to go with the "preds", i'm afraid unless you know how to exceed the lint limit to 90 characters.